File: 1. Beginner/bee-1043.py
A, B, C = map(float, input().split())

if ((A + B) > C and (B + C) > A and (C + A) > B):
    print(f"Perimetro = {round((A + B + C),1)}")
else:
    print(f"Area = {round((0.5 * (A + B) * C),1)}")

File: 1. Beginner/Golang/bee-1043.go
package main

import "fmt"

func main() {
	var A, B, C float64
	fmt.Scanf("%v %v %v", &A, &B, &C)

	if (A+B) > C && (B+C) > A && (C+A) > B {
		fmt.Printf("Perimetro = %.1f\n", (A + B + C))
	} else {
		fmt.Printf("Area = %.1f\n", (0.5 * (A + B) * C))
	}
}
